CHANGELOG.md

```diff
@@ -1,44 +1,96 @@
 # Change Log
-
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/) and this
-project adheres to [Semantic Versioning](http://semver.org/).
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+Extending the adopted spec, each change should have a link to its corresponding pull request appended.
 
 ## [Unreleased]
+
+## [v1.0.0] - Unreleased
+### Added
+* Allow creation of service accounts. [#80]
+* Add support for private clusters via submodule. [#69]
+* Add `remove_default_node_pool` set to `false` by default. Fixes [#15]. [#55]
```

Comment:
An explanation of what was fixed should go under Fixed.

Reply:
#15 was a feature request that #55 implemented; I've updated this line to indicate that it's implementing a feature.